webnode/src/config/index.js

```diff
@@ -1,15 +1,9 @@
 export const IS_DEV = process.env.NODE_ENV === "development";
 export const DEBUGGING = process.env.DEBUG;
 export const API_VERSION = "api/v2";
-export const API_ROOT_URL =
-  window.location.href.indexOf("localhost") > 0
-    ? "http://18.217.133.146:3000"
-    : "https://broker-1.oysternodes.com:3000";
-
-export const ASSET_URL =
-  window.location.href.indexOf("localhost") > 0
-    ? window.location.href
-    : "https://web.oysternode.com/";
+export const API_ROOT_URL = "https://broker-1.oysternodes.com:3000";
```

Comment:
It would be nice if this could randomly use broker-1 sometimes and broker-2 the other times, so it's not always the same broker that deals with the webnodes.

Reply:
Actually, now that I think about it, what was wrong with this?

```  window.location.href.indexOf("localhost") > 0	+
    ? "http://18.217.133.146:3000"	+export const ASSET_URL = "https://web.oysternode.com/";
    : "https://broker-1.oysternodes.com:3000";```   

And this?  

```export const ASSET_URL =	
  window.location.href.indexOf("localhost") > 0	
    ? window.location.href	
    : "https://web.oysternode.com/";```

We should probably keep these changes so devs don't have to manually change the file while QAing, unless we use a different config when we're in dev mode or something.

Reply:
I had troubles with window.location...  after use windows.location I couldnt use href for example :(
but I will try to do something with it

Reply:
I think you might to do a merge cause i jsut added some more error checking to fix tests.

```
export const API_ROOT_URL =
  (window.location && window.location.href.indexOf('localhost') > 0) > 0
    ? "http://18.217.133.146:3000"
    : "https://broker-1.oysternodes.com:3000";

export const ASSET_URL =
  (window.location && window.location.href.indexOf('localhost') > 0) > 0
    ? window.location.href
    : "https://web.oysternode.com/";
```

Reply:
fyi: During merge I will accept your changes but I have troubles with merging, thanks improves :-)).. but I will fix it